README.md

```diff
@@ -8,20 +8,18 @@ Our pipeline also provides utilities to export the models to ONNX format for use
 We will grow the collection of the models suitable for high-throughput imaging and phenotyping.
```

Comment:
This package contains a growing collection of models suitable for high-throughput imaging and phenotyping. 
Some applications include:
- robust virtual staining of cellular landmarks 
- infection classifier (WIP)

Reply:
Now moved to the top.